.github/dependabot.yml

```diff
@@ -17,3 +17,15 @@ updates:
     versions:
     - 0.8.1
     - 0.8.4
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  reviewers:
+  - devstein
+  assignees:
+  - devstein
+  labels:
+  - dependencies
+  - please code review
```

Comment:
I copy-pasted this config from the go setup, so there is a non-zero chance that things need some tweaking.